src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -22,6 +22,10 @@
      */
     String getDescription();
 
+    default String getHyphenatedName(){
```

Comment:
`StepView` would make sense since it's the only place that uses this to set a CSS class.

It looks like you also use the same value to set the ID in `OperationView`, so you'll also need some way to compute that.  I don't see any reason why this needs to be the same value as a CSS class used in `StepView`, so this can be a separate thing.